src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -109,7 +102,7 @@ public async Task InvokeAsync(HttpContext context)
                 var result = await executer.ExecuteAsync(
                     gqlRequest.OperationName,
                     gqlRequest.Query,
-                    gqlRequest.GetInputs(),
+                    gqlRequest.Inputs,
```

Comment:
Does the request have such a property?

Reply:
Nope. Sorry, keep going back and fwd between ideas here. It's not easy.